git-use-https/README.md

```diff
@@ -0,0 +1,26 @@
+# Use HTTPS for Git data transfer
+
+GitHub Action to use Git over HTTPS with OAuth instead of SSH to transfer data to GitHub. This fixes cloning private GitHub repositories, when unable to use `actions/checkout`.
```

Comment:
Is it possible to checkout private repos with `actions/checkout` after running this action, or do I need to use the classic `git clone https://...` afterwards?

Reply:
Just confirmed that it still is in https://github.com/greenbone/notus-generator/actions/runs/7759320870/job/21163111380